index.html

```diff
@@ -0,0 +1,976 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+        <link href="cssreset" rel="stylesheet" type="text/css" />
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous" />
+        <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@800&display=swap" rel="stylesheet" />
+        <link href="css/style.css" rel="stylesheet" />
+        <script src="https://kit.fontawesome.com/cbdf2b0f0e.js" crossorigin="anonymous"></script>
+        <title>News Week Magazine</title>
+    </head>
+    <body>
+        <div class="container-fluid header-main border clearfix top-header">
+            <!-- Mobile View-->
+            <nav class="navbar mobile-navbar p-0 clearfix navbar-expand-sm navbar-expand-md fixed-top d-lg-none navbar-light">
+                <div class="w-30 p-0 h-auto float-left">
+                    <a class="navbar-brand pt-0 pl-3 text-white" href="#">Newsweek</a>
+                </div>
+                <div class="w-70 float-right">
+                    <ul class="top-navbar mt-1 float-right pr-2">
+                        <li class="btn btn-sm text-white d-none d-sm-inline d-md-inline d-lg-block"><a href="#" class="text-white">Login</a></li>
+                        <li class="btn btn-sm d-sm-inline d-md-none d-lg-none"><a href="#"><i class="far fa-user text-white"></i></a></li>
+                        <li class="btn btn-sm bg-warning d-sm-inline d-md-inline rounded-pill"><a href="#" class="text-dark">Subscribe</a></li>
+                    </ul>
+                </div>
+            </nav>
+            <!-- Mobile View Ends Here-->
+            <header class="container-fluid header-container d-none d-lg-block">
+                <div class="page-header d-inline position-absolute">
+                    <h1 class="logo-text text-center text-white display-1">Newsweek</h1>
+                </div>
+                <div class="container-fluid">
+                    <div class="row">
+                        <div class="col-sm-6 mb-2 mt-5 p-0 text-white">Mon, Jul 20, 2020</div>
+                        <div class="col-sm-6 mb-2 mt-5 p-1 d-flex justify-content-end">
+                            <button class="btn text-white">Login</button>
+                            <button class="btn bg-warning rounded-pill">Subscribe</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="container-fluid weather">
+                    <div class="api">
+                        <a href="/weather" title="Weather" class="weather-cwidget ml-n3 text-white">
+                            <picture style="display: inherit;">
```

Comment:
- [ ] For good coding practices, don't use inline styling, put your styles in your CSS